sootup.tests/src/test/resources/validator/jimple/InvokeArgumentValidator.jimple

```diff
@@ -0,0 +1,77 @@
+public class InvokeArgumentValidator extends java.lang.Object
+{
+    public void <init>()
+    {
+        InvokeArgumentValidator l0;
+
+        l0 := @this: InvokeArgumentValidator;
+        specialinvoke l0.<java.lang.Object: void <init>()>();
+
+        return;
+    }
+
```

Comment:
add a testcase for array types. Primitive type array, Classtype array and both cases also as multi-dimensional array